chapters/data-analysis.tex

```diff
@@ -1,182 +1,236 @@
+and important step for documentation, collaboration and conflict-solving.
 Version control allows you to effectively track code edits,
-including the addition and deletion of files. 
-This way you can delete code you no longer need, 
+including the addition and deletion of files.
+This way you can delete code you no longer need,
 and still recover it easily if you ever need to get back previous work.
+Everything that can be version-controlled should be.
 Both analysis results and data sets will change with the code.
-You should have each of them stored with the code that created it.
-If you are writing code in Git/GitHub,
+Whenever possible, you should track have each of them with the code that created it.
+If you are writing code in Git or GitHub,
 you can output plain text files such as \texttt{.tex} tables
 and metadata saved in \texttt{.txt} or \texttt{.csv} to that directory.
 Binary files that compile the tables,
 as well as the complete data sets, on the other hand,
-should be stored in your team's shared folder. 
+should be stored in your team's shared folder.
 Whenever data cleaning or data construction codes are edited,
 use the master script to run all the code for your project.
-Git will highlight the changes that were in data sets and results that they entail. 
+Git will highlight the changes that were in data sets and results that they entail.
 
 %------------------------------------------------
 
-\section{Data cleaning}
-
-Data cleaning is the first stage of transforming the data you received from the field into data that you can analyze.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning}}
-The cleaning process involves (1) making the data set easily usable and understandable, 
-and (2) documenting individual data points and patterns that may bias the analysis.
-The underlying data structure does not change.
-The cleaned data set should contain only the variables collected in the field.
-No modifications to data points are made at this stage, except for corrections of mistaken entries.
+\section{De-identifying research data}
 
-Cleaning is probably the most time consuming of the stages discussed in this chapter.
-This is the time when you obtain an extensive understanding of  the contents and structure of the data that was collected.
-Explore your data set using tabulations, summaries, and descriptive plots.
-You should use this time to understand the types of responses collected, both within each survey question and across respondents.
-Knowing your data set well will make it possible to do analysis.
-
-\subsection{De-identification}
-
-The initial input for data cleaning is the raw data.
+The starting point for all tasks described in this chapter is the raw data.
 It should contain only materials that are received directly from the field.
 They will invariably come in a host of file formats and nearly always contain personally-identifying information.\index{personally-identifying information}
```

Comment:
```suggestion
Raw data will invariably come in a host of file formats and nearly always contain personally-identifying information, meaning that it will have to stay encrypted as described in the previous chapter.\index{personally-identifying information}
```

I found it weird this sentence starts with they, but if that is just me then no problem. I think we should make _one_ reference to safe storage of the raw data in the beginning of each section that describe safe practices for raw data